src/ocean/util/serialize/contiguous/package_.d

```diff
@@ -696,3 +697,46 @@ unittest
     auto d = Deserializer.deserialize!(S2)(buffer);
     test(deepEquals(*d.ptr, s));
 }
+
+/******************************************************************************
+
+    Const arrays of arrays
+
+******************************************************************************/
+
+struct ConstS
+{
+    Const!(int) n = 3;
+    Const!(char[][]) a = ["Hello", "World"];
+    Const!(char[])[2][3] b = [
+        ["Die", "Katze"], ["tritt", "die"], ["Treppe", "krumm."]
+    ];
+}
+
+unittest
+{
+    void[] buffer;
+
+    ConstS s;
+
+    Serializer.serialize(s, buffer);
+    auto cont_S = Deserializer.deserialize!(ConstS)(buffer);
```

Comment:
The motivation is to support struct fields containing an array of swarm channels (i.e. strings) where the original channels, passed by the user to the swarm API, can be `immutable(char[])[]`.